lighthouse-core/test/runner-test.js

```diff
@@ -789,6 +793,25 @@ describe('Runner', () => {
       expect(lhr.runtimeError.message).toBeDisplayString(/did not paint any content.*\(NO_FCP\)/);
     });
 
+
+    it('includes a crash runtimeError when there\'s a crash during gathering', async () => {
```

Comment:
I'd love to get a smoketest of this, but it's too hard without puppeteer. Maybe with FR support one can be added to the pptr test suite there? :D

Reply:
maybe doable. looks like it requires a 

```js
driver.gotoURL('chrome://crash');
```

you can't navigate to that URL clientside, so i think it's gotta come via protocol.

Reply:
yea this really could use a smoke test